pythonbytes.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+import os
+import sys
+import argparse
+from pathlib import Path
+import concurrent.futures
+import requests
+from lxml import etree
+
+
+RSS_URL = 'https://pythonbytes.fm/episodes/rss'
```

Comment:
Is this working without modification of the original classes? I can't see because it's not splitted between several commits.